hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson

```diff
@@ -385,7 +385,28 @@
       run_opts: ["+scb_otf_en=1", "+ecc_mode=1", "+en_always_all=1"]
       reseed: 5
     }
-  ]
+    {
+      name: flash_ctrl_phy_arb_redun
+      uvm_test_seq: flash_ctrl_phy_arb_redun_vseq
+      run_opts: ["+scb_otf_en=1", "+otf_num_rw=5", "+otf_num_hr=10",
+                 "+en_always_all=1", "+bypass_alert_ready_to_end_check=1"]
+      reseed: 3
```

Comment:
slightly prefer to use bigger reseed, unless we have very little randomization in the test.

Reply:
increase to 5